assets/panel/debugger.properties

```diff
@@ -1023,8 +1029,12 @@ shortcuts.toggleBreakpoint=Toggle Breakpoint
 shortcuts.toggleBreakpoint.accesskey=B
 
 # LOCALIZATION NOTE (shortcuts.toggleCondPanel): text describing
-# keyboard shortcut action for toggling conditional panel keyboard
-shortcuts.toggleCondPanel=Toggle Conditional Panel
+# keyboard shortcut action for toggling conditional panel for breakpoints
+shortcuts.toggleCondPanel=Edit Conditional Breakpoint
```

Comment:
Please see this document for making changes to existing strings
https://developer.mozilla.org/en-US/docs/Making_String_Changes

In short: you need a new string ID, and update all the code. I'm also not sure I understand how this fits into the PR subject, but that's more for debugger owners.

Reply:
Thanks for bringing this to my attention. I'll discuss viable solutions with the debugger owners.

Reply:
I appended a differentiating property to the end of `toggleCondPanel` between log points (`toggleCondPanel.logPoint`)and breakpoints (`toggleCondPanel.breakpoint`) so there is no change to existing strings. @flodolo these changes seem to be consistent with the localization best practices you linked. Let me know if this works